### PR TITLE
feat: Add data persistency and improve docs - Docker Compose

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
- Improve Docker Compose dev docs
- Add data persistency

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4100"></a>

#### Screenshot

Screenshot 2023-01-27 at 2 38 48 PM

### CODE REVIEW

File: pages/docs/environments/local-development-docker-compose.mdx [hunk 1]
```diff
@@ -3,17 +3,39 @@ import Link from "../../../components/Link";
...
-Full details coming soon.
+2) Execute the following command, this will build the images and start the containers.
+
+```bash
+docker compose up
+```
+
+3) You can now access the application at `http://localhost:3000`.
+
+4) If you making any chnages to the dependices, it's recommended to rebuild the images with the following command.
```
Comment (eddiejaoude) [inline pages/docs/environments/local-development-docker-compose.mdx:24]:
```suggestion
4) If you making any changes to the dependencies, it's recommended to rebuild the images with the following command. 
```

**Comment by eddiejaoude** [comment]
Looks good 👍  just 2 small typos

**Comment by Pradumnasaraf** [comment]
Thank you, Eddie, for the review. Fixed it.

**Comment by yashdev9274** [comment]
Hey @eddiejaoude and @Pradumnasaraf I would like to  contribute my Docker and Kubernetes skills on this project, where I can provide my help with this? Do let mw know

**Comment by Pradumnasaraf** [comment]
> Hey @eddiejaoude and @Pradumnasaraf I would like to contribute my Docker and Kubernetes skills on this project, where I can provide my help with this? Do let mw know

Hey, you can look at the config files and let's know how we can improve them. Feel free to raise an issue for that

**Pull request merged**